Resources/Prototypes/Loadouts/neck.yml

```diff
@@ -158,3 +158,216 @@
   exclusive: true
   items:
     - ClothingNeckGoldAutismPin
+
+# Bedsheets
+- type: loadout
+  id: LoadoutNeckBedsheetBlack
+  category: Neck
+  cost: 2
+  exclusive: true
+  items:
+    - BedsheetBlack
+  requirements:
+    - !type:CharacterDepartmentRequirement
+       departments:
+         - Civilian
+    - !type:CharacterDepartmentRequirement
+       inverted: true
+       departments:
+         - Command
```

Comment:
Combining two opposite department requirements like this doesn't actually work.
Standard CharacterDepartmentRequirement is essentially
"MUST BE ANY OF THESE DEPARTMENTS:"

Inverted CharacterDepartmentRequirement is
"CANNOT BE ANY OF THESE DEPARTMENTS"

So Requiring a Civilian department means that anyone *not* in a Civilian department can't have the item already, making the Inverse Command Requirement redundant.

Reply:
HoP is both part of the Civilian and Command department. When I remove the requirement to be not part of Command, HoP can still access these items as they are part of the Civilian department.